ivanti_connect_secure/assets/dashboards/ivanti_connect_secure_overview.json

```
@@ -0,0 +1,1528 @@
+                                            "data_source": "logs",
+                                            "search": {
+                                                "query": "source:ivanti-connect-secure $Message-ID $User-Name $Client-IP $Severity"
+                                            },
+                                            "indexes": [
+                                                "*"
+                                            ],
+                                            "group_by": [
+                                                {
+                                                    "facet": "@http.useragent_details.os.family",
+                                                    "limit": 10,
+                                                    "sort": {
+                                                        "aggregation": "count",
+                                                        "order": "desc",
+                                                        "metric": "count"
+                                                    }
+                                                }
+                                            ],
+                                            "compute": {
+                                                "aggregation": "count"
+                                            },
+                                            "storage": "hot"
+                                        }
+                                    ],
+                                    "response_format": "scalar",
+                                    "sort": {
+                                        "count": 500,
+                                        "order_by": [
+                                            {
+                                                "type": "formula",
+                                                "index": 0,
+                                                "order": "desc"
+                                            }
+                                        ]
+                                    },
+                                    "style": {
+                                        "palette": "datadog16"
+                                    }
+                                }
+                            ],
+                            "type": "sunburst",
+                            "legend": {
+                                "type": "table"
+                            }
+                        },
+                        "layout": {
+                            "x": 0,
+                            "y": 17,
+                            "width": 6,
+                            "height": 4
+                        }
+                    },
+                    {
+                        "id": 5107853845773138,
+                        "definition": {
+                            "title": "Count of Logs by User-Agents",
```

```suggestion
                            "title": "Count of Logs by User-Agent",
```
I'm not 100% sure on this one, please ignore if incorrect

---

Updated to `User-Agent`